libraries/AP_Mission/AP_Mission_Commands.cpp

```diff
@@ -219,7 +219,7 @@ bool AP_Mission::start_command_do_gimbal_manager_pitchyaw(const AP_Mission::Miss
     }
 
     // check gimbal device id.  0 is primary, 1 is 1st gimbal, 2 is 2nd gimbal, etc
-    uint8_t gimbal_instance = mount->get_primary();
+    uint8_t gimbal_instance = mount->get_primary_id();
```

Comment:
I'd prefer if we used "instance" instead of "id".  I worry that "id" could be misinterpreted as some unique id from the camera when really it is the instance within the driver array that we are getting back.

Reply:
.. or how about get_primary_index?

Reply:
> .. or how about get_primary_index?

Done.

It's not *quite* right in that I'd think of the instance being an instance of the backend class, but I'm not that fussed.

I considered actually grabbing an object back to play with here - but sanity prevailed in this case.

Reply:
yeah, that's why my second suggestion was "index" which makes it more clear that it is a number.. anyway, I think I'll approve this now, thanks!